weekly-activity.sh

```diff
@@ -1,13 +1,17 @@
 #! /bin/bash
 set -e
 
-starting_date=$(date -d "7 days ago" +"%Y-%m-%d")
+which gdate > /dev/null && date_cmd="gdate" || date_cmd="date"
 
-# PRs
-gh search prs --author "@me" --created ">=$starting_date"
+starting_date=$($date_cmd -dlast-monday +"%Y-%m-%d")
+gh_search_flags="--author @me --updated >=$starting_date --json repository,title,url"
```

Comment:
```suggestion
gh_search_flags="--updated >=$starting_date --json repository,title,url"
```

For reviews, we need to use `--reviewed-by @me` instead of the author flag. So probably need to leave that configurable for each of the different runs.

Reply:
nice catch, must've lost that in my copy-pasting. feel free to commit the suggested updates